lib/trubl/base.rb

```diff
@@ -28,7 +28,9 @@ def missing_or_exception?(response)
           nil
         end
 
-        code && (400..600).include?(code)
+        is_missing_or_exception = code && (400..600).include?(code)
```

Comment:
I think that's a good idea, I'll give it a shot if it makes sense based on the overall strategy rethink we discussed with @tommyh yesterday.